README.md

```diff
@@ -11,18 +11,14 @@ The site chrome are all in JS files in `site/_core/`.
 
 The first time, get all the dependencies loaded via
 
-```
-npm install
-```
+`$ npm install` or `$ yarn install`
```

Comment:
I think we should discuss with the community to specify only a single package manager for coherency in builds and runs. 
What would you suggest @orta ?

Reply:
There's a yarn lockfile,  and that's what's on CI - it's a yarn project, but it's fine if people want to dev with NPM. It'll be deterministic in the way that matters. 👍